modules/nf-core/seqfu/derep/tests/main.nf.test

```diff
@@ -0,0 +1,65 @@
+nextflow_process {
+
+    name "Test Process SEQFU_DEREP"
+    script "../main.nf"
+    process "SEQFU_DEREP"
+
+    tag "modules"
+    tag "modules_nfcore"
+    tag "seqfu"
+    tag "seqfu/derep"
+
+    test("derep - 3 fastas") {
+ 
+        when {
+            process {
+                """
+                input[0] = [
+                            [ id:'test' ],
+                            [
+                                file(params.test_data['sarscov2']['genome']['proteome_fasta'], checkIfExists: true),
+                                file(params.test_data['sarscov2']['genome']['genome_fasta'], checkIfExists: true),
+                                file(params.test_data['sarscov2']['genome']['genome_fasta_gz'], checkIfExists: true)
+                            ]
+                        ]
+                """
+            }
+        }
+
+        then {
+            assertAll(
+                { assert process.success },
+                { assert snapshot(process.out).match() },
```

Comment:
This is checking that **all** the output channels give consistent output, the md5sum is in the snap file. So don't need the other assertions